lib/Doctrine/DBAL/Configuration.php

```diff
@@ -40,11 +41,15 @@ public function setSQLLogger(SQLLogger $logger = null)
     /**
      * Gets the SQL logger that is used.
      *
-     * @return \Doctrine\DBAL\Logging\SQLLogger|null
+     * @return \Doctrine\DBAL\Logging\SQLLogger
```

Comment:
This should be dropped.

Reply:
Why remove the `@return` statement? Is there a coding standard that Doctrine follows? I tried to propose one a few years ago, but this lead nowhere at the time.

Reply:
Yes, Doctrine projects use [Doctrine Coding Standard](https://github.com/doctrine/coding-standard), it has been improved a lot in recent months.
This is redundant and useless information (there is already native type), thus should be removed.

Reply:
Thanks for the pointer, I missed it! Removed `@return` then.